lib/Modules/FreeRdpHooks/FreeRdpHooks.psm1

```diff
@@ -24,33 +24,26 @@ Import-Module JujuHooks
 Import-Module OpenStackCommon
 
 function Get-Vcredist {
-    BEGIN {
+    Write-JujuWarning "Getting vcredist."
+    $vcredistUrl = Get-JujuCharmConfig -Scope "vcredist-url"
+    if(!$vcredistUrl) {
         $resourcesSupport = (Get-Command resource-get.exe -ErrorAction SilentlyContinue) -ne $null
```

Comment:
I don't want to fall back if the `Get-JujuResource` didn't work for some other reasons (network failures, resource not attached, etc). Fallback should be used only when Juju resources are not supported.